runhouse/utils.py

```diff
@@ -544,3 +547,53 @@ def format(self, output_type):
             self._display_title = True
 
         return system_color, reset_color
+
+
+####################################################################################################
+# Status collection utils
+####################################################################################################
+class ServletType(str, Enum):
+    env = "env"
+    cluster = "cluster"
+
+
+def get_gpu_usage(collected_gpus_info: dict, servlet_type: ServletType):
+
+    gpus = list(collected_gpus_info.keys())
```

Comment:
```suggestion
    gpu_device_indices = list(collected_gpus_info)
```

is this what the keys represent?

Reply:
The keys are actually the GPU indexes (some sort of id)
![image](https://github.com/user-attachments/assets/fb96f0f4-527a-4a0c-8d9a-fa68d632e795)